README.md

```diff
@@ -8,23 +8,26 @@
 [travis-badge]: https://travis-ci.org/picostyle/picostyle.svg
 [travis]: https://travis-ci.org/picostyle/picostyle
 
-<div align="center">
-  <a href="https://github.com/morishitter/picostyle">
-    <img width="360px" src="http://morishitter.github.io/picostyle.svg">
-  </a>
-</div>
-<br>
+<a href="https://github.com/morishitter/picostyle">
+  <img width="20%" src="http://morishitter.github.io/picostyle.svg">
+</a>
+
+Picostyle is a 0.3 KB CSS-in-JS library for use with frameworks that expose an `h` function.
+
+Currently tested with:
 
-Picostyle is a 0.3 KB CSS-in-JS library for use with [Hyperapp](https://github.com/hyperapp/hyperapp) & [Picodom](https://github.com/picodom/picodom).
+- [Hyperapp](https://github.com/hyperapp/hyperapp)
+- [Picodom](https://github.com/picodom/picodom)
+
+<br>
 
-[Try it Online](https://codepen.io/morishitter/pen/qXaPYQ?editors=0010)
+[Try it Online](https://codepen.io/lukejacksonn/pen/xXRGpv?editors=0010)
 
 ## Features
 
 - **🚀 The smallest CSS-in-JS library**: Only 0.3 KB (minified & gzipped).
 - **👏 Zero dependencies**: And under 60 LOC.
 - **💅 Styled components**: Gives you a styled component like [styled-components](https://www.styled-components.com/) that y'all love.
-- **❤️ For Hyperapp & Picodom**: The 1 KB frontend library family.
 
 ## Installation
```

Comment:
Maybe add a section to explain how to grab from unpkg, jsDelivr, etc. Just like we do in Hyperapp, etc.

Reply:
Copied the one from hyperapp :+1: